pytorch/buildspec-1-5-eia.yml

```diff
@@ -29,9 +29,9 @@ images:
     build: &PYTORCH_CPU_INFERENCE_PY3 false
     image_size_baseline: 6318
     device_type: &DEVICE_TYPE cpu
-    python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py36
-    os_version: &OS_VERSION ubuntu16.04
+    python_version: &DOCKER_PYTHON_VERSION py38
+    tag_python_version: &TAG_PYTHON_VERSION py38
```

Comment:
question - why are we editing buildspec-1-5-eia.yml instead of buildspec-eia.yml?